frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.tsx

```diff
@@ -1874,7 +1878,7 @@ class SegmentsView extends React.Component<Props, State> {
     if (visibleSegmentationLayer == null) return false;
     const relevantSegments =
       groupId != null ? this.getSegmentsOfGroupRecursively(groupId) : this.getSelectedSegments();
-    if (relevantSegments == null) return false;
+    if (relevantSegments == null || relevantSegments?.length === 0) return false;
```

Comment:
```suggestion
    if (relevantSegments == null || relevantSegments.length === 0) return false;
```

The second part of the expression is only evaluated when relevantSegments is not null.